src/pages/EditMonitor.vue

```diff
@@ -930,6 +931,9 @@ message HealthCheckResponse {
   </soap:Body>
 </soap:Envelope>` ]);
             }
+            if (this.monitor && this.monitor.httpBodyEncoding && this.monitor.httpBodyEncoding === "form") {
```

Comment:
```suggestion
            if (this.monitor && this.monitor.httpBodyEncoding === "form") {
```

Does the middle condition serve any purpose? I know that XML also has it so we might want to keep it in here for uniformity, but does it actually do anything useful?

Reply:
I think it could be simpler.